course/assignments/reproducible-research-assignment.md

```diff
@@ -0,0 +1,21 @@
+## Reproducible Research Plan:
+
+Based on the [Data Reuse Plan](https://github.com/mozillascience/working-open-workshop/blob/gh-pages/handouts/data_reuse_plan_template.md) by Mozilla WOW.  
+
```

Comment:
Given this is an R project, there are some standard files that are included that complete some of these requirements. So maybe say: "Using the tools that were covered, setup a new project for your research project (the one for your final assignment). Make sure that the follow files and folders were created after setting up the project: DESCRIPTION, README.Rmd (or md), LICENSE, the .Rproj file, data/ (folder), R/ (folder), doc/ (folder)."